kuma/wiki/tests/test_admin.py

```diff
@@ -54,6 +55,30 @@ def test_slug_short(self):
         dsa.slug = 'Web/A_long_slug_that_will_be_truncated'
         assert self.admin.slug_short(dsa) == 'Web/A_long_slug_that_w...'''
 
+    def test_doc_short_without_document(self):
+        dsa = DocumentSpamAttempt(slug='Slug')
+        assert self.admin.doc_short(dsa) == '<em>new document</em>'
+
+    def test_doc_short_with_document(self):
+        slug = 'NotSpam'
+        html = '<p>This page is not spam.</p>'
+        doc = document(title='blah', slug=slug, html=html, save=True,
+                       locale='en_US')
```

Comment:
I know the test will pass nevertheless, but to prevent red herrings in locale parsing, I'd use `en-US` here